queue/src/fiskaltrust.Middleware.Storage.SQLite/DatabaseInitialization/DatabaseMigrator.cs

```diff
@@ -74,6 +74,22 @@ public async Task MigrateAsync()
                 await SetCurrentVersionAsync(connection, Path.GetFileNameWithoutExtension(migrationScript)).ConfigureAwait(false);
                 _logger.LogDebug($"Applying the migration script was successful. Set current version to {Path.GetFileNameWithoutExtension(migrationScript)}.");
             }
+
+            if (await DoesFtJournalFRCopyPayloadTableExist(connection))
+            {
+                _logger.LogInformation("Migration was successful, the table ftJournalFRCopyPayload exists.");
+            }
+            else
+            {
+                _logger.LogError("Migration failed, the table ftJournalFRCopyPayload does not exist.");
+            }
+        }
+        
+        private async Task<bool> DoesFtJournalFRCopyPayloadTableExist(IDbConnection connection)
+        {
+            var tableName = await connection.ExecuteScalarAsync<string>("SELECT name FROM sqlite_master WHERE type='table' AND name='ftJournalFRCopyPayload'").ConfigureAwait(false);
+            return tableName != null;
+            
```

Comment:
I like the idea 😁 but as we also don't check that for the other migrations we don't need to check that here 🤔 I think this will get unmaintainable if we introduce that and do it for all future migrations